crates/swc_cli/src/commands/compile.rs

```diff
@@ -259,6 +291,8 @@ impl super::CommandRunner for CompileOptions {
         }
 
         if let Some(stdin_input) = stdin_input {
+            let span = tracing::span!(tracing::Level::TRACE, "compile_stdin");
+            let _ = span.enter();
```

Comment:
This will immediately drop

Reply:
thanks for catch, I forgot to update this along with non-stdin input spans.